src/nos/translators/inbound/event_sub.py

```diff
@@ -141,3 +142,40 @@ async def _consume_validated(
                     await self._handle_all_ivas_reset(payload=payload)
                 else:
                     await self._handle_iva_state_change(payload=payload)
+
+
+class OutboxSubTranslatorConfig(BaseSettings):
+    """Config for the outbox subscriber"""
+
+    user_data_event_topic: str = Field(
+        default=...,
+        description="The name of the topic containing user data events.",
+        examples=["user_data_events"],
+    )
+
```

Comment:
I made the default change for the users topic, but now I'm thinking that it might be easier for misconfiguration by omission (plus, defaults must be aligned between services or else they're pointless). Fixed the naming though in [27888e4](https://github.com/ghga-de/notification-orchestration-service/pull/6/commits/27888e4c1447a2ed270f818e6e6a0455adff1986)